erpnext/manufacturing/doctype/bom/bom.js

```diff
@@ -536,14 +542,20 @@ erpnext.bom.calculate_op_cost = function(doc) {
 	doc.operating_cost = 0.0;
 	doc.base_operating_cost = 0.0;
 
-	for(var i=0;i<op.length;i++) {
-		var operating_cost = flt(flt(op[i].hour_rate) * flt(op[i].time_in_mins) / 60, 2);
-		var base_operating_cost = flt(operating_cost * doc.conversion_rate, 2);
-		frappe.model.set_value('BOM Operation',op[i].name, "operating_cost", operating_cost);
-		frappe.model.set_value('BOM Operation',op[i].name, "base_operating_cost", base_operating_cost);
+	if(doc.with_operations) {
+		op.forEach((item) => {
```

Comment:
No need of op, use doc.operations.forEach

Reply:
remove the code var op = doc.operations || [];

Reply:
var op = doc.operations || []; declare on line 541 so that's why used otherwise we need to remove this line 541

Reply:
I know this was added by the team but you can remove